packages/e2e/docker-compose.yml

```
@@ -113,6 +108,7 @@ services:
   stake-pool-projector:
     environment:
       METADATA_JOB_RETRY_DELAY: 60
+      POOLS_METRICS_INTERVAL: '50'
```

For consistency with the existing number in line above
```suggestion
      POOLS_METRICS_INTERVAL: 50
```

---

Thank you. 63c303f3e7 should fix.